### ISSUE TITLE
Anchor map title boxes when scrolling

### ISSUE THREAD

**Issue by oevans** [Esri/Compare-Maps-Template#6]
When scrolling down in the side panel, the map title bar for the first map immediate scrolls up under the app description section (see first screenshot).  This doesn't provide a good separation or indication that you are still looking at the first web map description.

![screen shot 2014-12-11 at 11 15 17 am](https://cloud.githubusercontent.com/assets/3215530/5397289/59113202-8127-11e4-83a6-a85d7ab304e4.png)
![screen shot 2014-12-11 at 11 15 30 am](https://cloud.githubusercontent.com/assets/3215530/5397290/5b715298-8127-11e4-8b59-b43d3dbb0329.png)

I think having each map title bar anchor at the top, right under the app description section, would be an improvement. I'm envisioning the same way sections scroll by in iOS apps.  As sections scroll by the title of the current section anchors at the top until a new title section gets to the top and that replaces the one above it.

**Comment by kellyhutchins** [comment]
Thanks @oevans  we've made some updates to the UI for this template. At the next release (late March) we've added the description and legend to the map area.  This existing version will be archived so any apps based on this will still work but new apps will have the new look. 

If you want to submit a pull request for the update you suggest I'd be happy to merge it so that this version of the app is more usable.
